workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/ViewRegistry.kt

```diff
@@ -137,6 +131,7 @@ public fun <RenderingT : Any>
   @Suppress("UNCHECKED_CAST")
   return getFactoryFor(rendering::class)
     ?: (rendering as? AndroidViewRendering<*>)?.viewFactory as? ViewFactory<RenderingT>
+    ?: (rendering as? Named<*>)?.let { NamedViewFactory as ViewFactory<RenderingT> }
```

Comment:
I just realized this wrapping pattern (even before this change) might introduce an extra view unnecessarily when the wrapped rendering's factory is a ComposeViewFactory. It shouldn't affect correctness but something we should probably check/test and to keep in mind as yall adopt compose. 

Filed: https://github.com/square/workflow-kotlin/issues/546